### DISCUSSION TITLE
Promoting

### DISCUSSION THREAD

**Discussion by noraj** [mBouamama/rawsec_cli#60] [Show and tell]
I listed the CLI tool on the official "features" page and gave some examples : https://inventory.raw.pm/features.html#cli

**Comment by tyki6** [comment]
Wow amazing Job, thanks a lot

**Comment by noraj** [comment]
![image](https://user-images.githubusercontent.com/16578570/124320853-f2bf8f00-db7c-11eb-9407-5a8f6b146235.png)

Could you change the link to https://inventory.raw.pm/ please @mBouamama

**Reply by tyki6** [reply]
![image](https://user-images.githubusercontent.com/57527739/124458102-e27dfe80-dd8c-11eb-916c-4f2f52bec2c1.png)
Done
